sdk/core/Azure.Core/README.md

```diff
@@ -1,45 +1,100 @@
 # Overview 
```

Comment:
Missing title - Azure Core client library for .NET

Reply:
It's not a client library, so I did "Azure.Core shared library for .NET"